src/Middlewares/Middleware.php

```diff
@@ -93,14 +93,17 @@ function (ResourceContainer $container) {
                 $categoryService = pluginApp(CategoryService::class);
                 $isCategoryPage = $categoryService->getCurrentCategory() !== null && $this->activeOnCatPage;
                 $isInSearchOrCategoryPage = $this->isSearchPage || $isCategoryPage;
+                $currentCategory = $categoryService->getCurrentCategory();
 
-                if ($isInSearchOrCategoryPage && !$this->searchService->aliveTest()) {
-                    return false;
+                $showCategoryFilter = true;
+
+                // Show category filter only in the 0 and 1 level categories
+                if ($currentCategory !== null && $currentCategory->level > 1) {
+                    $showCategoryFilter = false;
                 }
 
-                if ($this->pluginConfig->get(Plugin::CONFIG_LOAD_NO_UI_SLIDER_STYLES_ENABLED)) {
```

Comment:
Some points here:
- I forgot that this configuration exists, so we will probably need to check it, when importing the CSS within the components. But I am not sure, if that works that easily, as it is probably still ending up within the plugin build.
    - Not sure if that counts as hacky, but we can import it under a `fl-no-ui-slider` class and only add this class if the configuration is set accordingly: `.fl-no-ui-slider { @import 'no-ui-slider.css' }`
- Maybe it would make sense to move all of the configurations into one object, that can be forwarded as one prop. This way we could also create a type for it